client/src/views/files/FoldersPage.vue

```diff
@@ -236,8 +242,10 @@ onUnmounted(async () => {
   routeWatchCancel();
 });
 
-async function onFileImportState(state: ImportState, data: StateData): Promise<void> {
-  console.log(state, data);
+async function onFileImportState(_state: ImportState, _importData?: ImportData, _stateData?: StateData): Promise<void> {
+  if (_state === ImportState.FileAdded) {
+    await fileUploadModal?.dismiss();
```

Comment:
This is going to be called for each added file. Also why do we need to dismiss it here too?

Reply:
done